packages/pass-style/src/types.d.ts

```diff
@@ -78,7 +79,9 @@ export type PassByCopy = Atom | Error | CopyArray | CopyRecord | CopyTagged;
 
 export type PassByRef =
   | RemotableObject
+  | RemotableBrand<any, any>
```

Comment:
What does `RemotableBrand<any, any>` cover that `RemotableObject` does not?

I understand the old definition here as corresponding to what this means dynamically. Looking at the new definition, all I infer is "Oh that's probably some magical type stuff that besides the point of describing the dynamic meaning of `PassByRef`"